crates/compiler/module/src/called_via.rs

```diff
@@ -1,7 +1,79 @@
+use self::Associativity::*;
 use self::BinOp::*;
 use std::cmp::Ordering;
 use std::fmt;
 
+const PRECEDENCES: [(BinOp, u8); 20] = [
```

Comment:
would it be worse to merge these tables? then the order for all of them is always the same. it's a bit messier visually I guess?

Reply:
Yeah also it feels anti-SoA 😄

Reply:
that does not matter; these are only traversed at compile time

Reply:
Fair point. Regardless, doesn't really seem necessary to combine them - I added tests verifying that all 3 have the same order as the enums they're based on (that is, for each entry in them, the `BinOp` in the tuple has the same underlying `u8` as the current index in the array)